chef_master/source/resource_ifconfig.rst

```diff
@@ -29,6 +29,7 @@ The full syntax for all of the properties that are available to the **ifconfig**
      bootproto                  String
      device                     String
      ethtool_opts               String
+     family                     String # defaults to 'inet' if not specified
```

Comment:
We should leave the default for the description below

Reply:
Oh, I did not catch this comment. It can be corrected after merge.